modules/gapi/misc/python/test/test_gapi_sample_pipelines.py

```diff
@@ -33,7 +33,7 @@ def test_mean_over_r(self):
         comp = cv.GComputation(g_in, g_out)
 
         for pkg in pkgs:
-            actual = comp.apply(in_mat, args=cv.compile_args(pkg))
+            actual = comp.apply(cv.gin(in_mat), args=cv.compile_args(pkg))
```

Comment:
in this required now?

Reply:
There is no another overload in python, only which takes `GRunArgs`